_internals/configuration.md

```diff
@@ -101,15 +101,15 @@ Here are the TDS related settings:
 - `babelfishpg_tds.unix_socket_directories`
 - `babelfishpg_tds.unix_socket_group`
 - `babelfishpg_tds.unix_socket_permissions`
-`
-Let us discuss these parameters in broader detail.
 
-### babelfishpg_tsql.allow_antlr_to_unsupported_grammar_for_testing
+Let us discuss these parameters in more detail.
+
+### `babelfishpg_tsql.allow_antlr_to_unsupported_grammar_for_testing`
 
 Variable for internal testing - make `antlr` allow some unsupported grammar.
 The default value is `false`.
```

Comment:
what are the risks when changing this value?

Reply:
You tell me.